### ISSUE TITLE
Add a disclaimer page

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#265]
- [x] I've confirmed that this document is not already in the [Docs Project](https://github.com/orgs/wpaccessibility/projects/3)

**Title**
Disclaimer

**Description**
The WP Accessibility Knowledge base is documentation to help WordPress users and site creators to make their work as accessible as possible. These the topics in this documentation are not the official guidelines to meet WCAG and cannot be legally used to refer to as official guidelines for web accessibility.

**Reason for Adding**
We need to document that the topics WP Accessibility Knowledge base are not the official guidelines to meet WCAG and cannot be used as legal material to prove a point. We need to find someone with legal expertise to help us write a solid text.

The link to this disclaimer can go in the main menu and in the home page for example, or on another spot that is easy discoverable.
